_posts/2021-06-25-feature-highlight-reporting.markdown

```diff
@@ -0,0 +1,79 @@
+
+**How can reports be generated ?**
+Reports are generated manually on OpenSearch Dashboards, via scheduled triggers, or through system triggers like alerts or API calls.
+
+**Anatomy of the Reporting feature on OpenSearch**
+The reporting feature is composed of two plugins; an [OpenSearch Dashboards plugin](https://github.com/opensearch-project/dashboards-reports/tree/main/dashboards-reports) which controls most the [user experience](https://github.com/opensearch-project/dashboards-reports/blob/main/docs/dashboards-reports/ux/OpenSearch-Dashboards-Reporting-UX-documentation.md) and the [OpenSearch plugin](https://github.com/opensearch-project/dashboards-reports/tree/main/reports-scheduler) which provides scheduling features and secures access to user data with OpenSearch Security policies. Our initial request for comments (RFC) for Reporting can be found [here](https://github.com/opensearch-project/dashboards-reports/blob/main/docs/dashboards-reports/dev/OpenSearch-Dashboards-Reporting-Design-Proposal.md).
+
+![Report Anatomy](/assets/media/blog-images/2021-06-25-feature-highlight-reporting/report-anatomy.png){: .img-fluid}
+
+## Quick Start User Guide
+
+### Installation
+
+Reporting comes packaged in with the OpenSearch and OpenSearch Dashboards downloads. You can follow the installation instructions on [opensearch.org/downloads](http://opensearch.org/downloads) 
+
+### How do I create a report?
+
+**Visual reports** are created inside the Visualization or Dashboards experience. 
+
+![visual reports](/assets/media/blog-images/2021-06-25-feature-highlight-reporting/visual-reports.gif){: .img-fluid}
+
+**CSV reports** are created from the Discover tab.
+
+![csv reports](/assets/media/blog-images/2021-06-25-feature-highlight-reporting/csv-reports.gif){: .img-fluid}
+
+**Scheduled reports** are configured from inside the Reporting UI under OpenSearch Plugins.
+
+![scheduled reports](/assets/media/blog-images/2021-06-25-feature-highlight-reporting/create-report-definition.gif){: .img-fluid}
+
+## Development internals
+
+The reporting feature uses a custom [minimal build](https://github.com/opensearch-project/dashboards-reports/tree/main/dashboards-reports/rendering-engine/headless-chrome) of headless-chromium. Chromium is used as the rendering engine to generate visual reports. Below is are multiple flow diagrams for the various actions of Reporting. 
+
+**Creating a report definition**: When a user defines a report in OpenSearch Dashboards, the Reporting Dashboards plugin saves that definition in an index in OpenSearch. 
+
+**Downloading a report:** When a user downloads a report, the OpenSearch Dashboards Reporting plugin fetches the data from OpenSearch. After the data is fetched, the report is generated and downloaded to the users client.
```

Comment:
What is 'client' in this context? Browser?

Reply:
fixed